react/fields/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.label {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    font-weight: @sk-bold;
+  }
+}
+
+.secondaryLabel {
+  font-weight: @sk-light;
```

Comment:
Has this been validated by the design team? Is a `secondaryLabel` the same weight regardless of screen size, even though the `label` is changing weight?